src/ScottPlot4/ScottPlot.Avalonia/AvaPlot.axaml.cs

```diff
@@ -56,27 +52,18 @@ public partial class AvaPlot : UserControl, ScottPlot.Control.IPlotControl
         /// </summary>
         public event EventHandler PlottableDragged;
 
-        [Obsolete("use 'PlottableDragged' instead", error: true)]
```

Comment:
Sorry I didn't catch these earlier, but I think we need to preserve these events.

Reply:
Is there a reason why the events should remain?
Since they are never called, this could cause a lot of confusion for users of the library. Thats was my intention to remove them completely.
But if necessary, I could add them like they where again ...

Reply:
They're part of the public API, we don't use them but our users might.

Reply:
With "using", you mean users are subscribing to it? Since the event is never called / fired it still wouldn't make sense to keep it.

Reply:
Those obsolete decorators are there to help guide people who are upgrading from very old versions of ScottPlot. Without the obsolete message, these users' code would break with no indication of how to fix it. Messages like this provide a path for them to get their code working again.

I'll stick the messages back in now and merge this PR in the next few minutes! 👍

Reply:
Oh, `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` made that harder than it needed to be lol

I'll try to fix that, then merge now